src/Dapr.Actors/Resources/SR.resx

```diff
@@ -179,7 +179,7 @@
     <value>The {0} interface '{1}' is using generics. Generic interfaces cannot be remoted.</value>
   </data>
   <data name="ErrorRemotedMethodCancellationTokenOutOfOrder" xml:space="preserve">
-    <value>Method '{1}' of {0} interface '{2}' has '{4}' parameter '{3}', and it is not the last parameter. If a method of the {0} interface has parameter of type '{4}' it must be the last parameter.</value>
+    <value>Method '{1}' of {0} interface '{2}' has parameter '{3}', while another argument was already of type '{4}'. If a method of the {0} interface has parameter of type '{4}', it must be the last parameter.</value>
```

Comment:
I don't think this is clearer - especially the `while another argument was already of type...` part.

Based on your example:

For example:

```C#
internal interface ITestActor : IActor
{
    Task Method(CancellationToken cancellationToken, bool additionalArgument);
}
```

I'd suggest the following:

> Method 'Method' of actor interface '...ITestActor' has a '...CancellationToken' parameter that is not the last parameter. If an actor method accepts a  '...CancellationToken' parameter then it must be the last parameter.

Reply:
Updated